build.sbt

```diff
@@ -15,22 +15,19 @@ ThisBuild / developers := List(
   tlGitHubDev("djspiewak", "Daniel Spiewak")
 )
 
-lazy val root = project
-  .in(file("."))
-  .enablePlugins(NoPublishPlugin)
-  .aggregate(
-    kernel,
-    noPublish,
-    settings,
-    github,
-    versioning,
-    mima,
-    sonatype,
-    ciSigning,
-    sonatypeCiRelease,
-    ci,
-    core,
-    ciRelease)
+lazy val root = tlCrossRootProject.aggregate(
+  kernel,
+  noPublish,
+  settings,
+  github,
+  versioning,
+  mima,
+  sonatype,
+  ciSigning,
+  sonatypeCiRelease,
+  ci,
+  core,
+  ciRelease)
```

Comment:
Here it is in action. Nothing fancy. Regardless of the name of the `val` it's assigned to, it always generates projects called `root`—I didn't want to fuss with macros.